src/main/resources/db/changelog/db.changelog-master.yaml

```
@@ -1,3 +1,5 @@
 databaseChangeLog:
   - include:
-      file: db/changelog/changes/01-create-books-table.yaml
\ No newline at end of file
+      file: db/changelog/changes/01-create-books-table.yaml
+  - include:
+      file: db/changelog/changes/02-create-users-table.yaml
```

```suggestion
      file: db/changelog/changes/02-create-users-table.yaml
      
```
Leave one empty line at the end of the file